gazebo_plugins/CMakeLists.txt

```diff
@@ -366,12 +366,24 @@ install(DIRECTORY test
 # in parallel.
 if (CATKIN_ENABLE_TESTING)
   find_package(rostest REQUIRED)
+
   add_rostest_gtest(set_model_state-test
                     test/set_model_state_test/set_model_state_test.test
                     test/set_model_state_test/set_model_state_test.cpp)
-  add_rostest(test/range/range_plugin.test)
   target_link_libraries(set_model_state-test ${catkin_LIBRARIES})
 
+  add_rostest(test/range/range_plugin.test)
```

Comment:
AFAIK they need a DISPLAY for creating the rendering. @iche033 could you please confirm it?